tools/python/requirements.txt

```diff
@@ -2,4 +2,6 @@ jsonschema==2.6.0
 pytest==3.5.0
 requests==2.18.4
 simplejson==3.13.2
-docopt==0.6.2
\ No newline at end of file
+docopt==0.6.2
+nexus-sdk; 
+python_version >= '3.5';
```

Comment:
This will break installation from pypi for Python<3.5. The solution has to work with the following install command:
```
pip install -r requirements.txt
```
For Python<3.5.
The nexus line should be written as:
`nexus-sdk; python_version >= '3.5'`